observable_test.go

```diff
@@ -618,6 +618,29 @@ func TestObservableDistinctUntilChanged(t *testing.T) {
 	assert.Exactly(t, []int{1, 2, 1, 3}, nums)
 }
 
+func TestObservableSequenceEqualWithCorrectSequence(t *testing.T) {
+	sequence := Just(2, 5, 12, 43, 98, 100, 213)
+	result := Just(2, 5, 12, 43, 98, 100, 213).SequenceEqual(sequence)
+	AssertThatSingle(t, result, HasItems(true))
```

Comment:
`HasValue` is the assertion to be used with `Single`

Reply:
For the time being the assertion API is not typed, this might be improved at some point.